x-pack/plugins/endpoint/public/applications/endpoint/store/policy_list/selectors.ts

```diff
@@ -4,10 +4,23 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { PolicyListState } from '../../types';
+import { createSelector } from 'reselect';
+import { PolicyListState, PolicyDetailsState } from '../../types';
 
 export const selectPolicyItems = (state: PolicyListState) => state.policyItems;
 
+export const selectPolicyDetails = (state: PolicyDetailsState) => state.policyItem;
+
+export const selectPolicyIdFromParams: (state: PolicyDetailsState) => string = createSelector(
```

Comment:
Just gets the job done right now, we should break out the `location` state to another location in state as discussed in previous threads by @paul-tavares and @peluja1012 
https://github.com/elastic/kibana/pull/58308/files#r385053154

Then we could also put selectors like this in a more generic place